classes/requests/post/class-kp-unavailable-features.php

```diff
@@ -68,10 +68,21 @@ protected function get_request_url() {
 	protected function get_body() {
 		return array(
 			'installation_data' => array(
+				'platform_data'      => array(
+					'platform_name'        => 'woocommerce',
+					'platform_version'     => WC()->version,
+					'platform_plugin_name' => 'woocommerce_klarna_payments',
+				),
 				'klarna_plugin_data' => array(
-					'plugin_identifier' => 'klarna:plugins:woocommerce:klarna-plugin',
-					'plugin_version'    => '3.8.3',
+					'plugin_identifier' => 'woocommerce_klarna_payments',
+					'plugin_version'    => WC_KLARNA_PAYMENTS_VERSION,
+				),
+				'store_data'         => array(
+					'store_urls' => array(
+						'base_url' => get_site_url(),
+					),
```

Comment:
In the request from Klarna this was just a array of strings, not a key/value pair.